benchmarks/README.md

```diff
@@ -5,7 +5,30 @@ This directory contains benchmarking schemas and data sets that you
 can use to test a variety of performance scenarios against your
 protobuf language runtime.
 
-The schema for the datasets is described in `benchmarks.proto`.
+## Benchmark tools and build instructions
+
+First, you need to follow the instruction in the root directory's README to 
+build your language's protobuf, then:
+
+### CPP
+We are using [google/benchmark](https://github.com/google/benchmark) as the 
+benchmark tool for testing cpp. This is included as submodule under third_party
+directory. To init and build this tools, you need to do this under root dirctory:
+
+```
+$ cd third_party
+$ git submodule update --init -r
+$ cd benchmark && cmake -DCMAKE_BUILD_TYPE=Release && make && cd ../..
```

Comment:
Can you include these as part of `make cpp`?

Reply:
Done